recipes/cupy/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export NCCL_LIB_DIR="${PREFIX}/lib"
+export NCCL_INCLUDE_DIR="${PREFIX}/include"
```

Comment:
Should these go in `nccl` package as an activate script? If so, can you open an issue in nccl feedstock?

Reply:
I don't think so. These are environment variables that [some builds use (like `pytorch`)]( https://github.com/pytorch/pytorch/search?q=NCCL_LIB_DIR&unscoped_q=NCCL_LIB_DIR ). NCCL itself doesn't use them. That said, these might not be needed here any more. I'll check.